### PR TITLE
Translation: Update Spanish.Properties

### PR DESCRIPTION
**Pull request by ArchDuque-Pancake**
Mosaicos? Pero si yá está el estandar de "Casilla"?...

### CODE REVIEW

**Comment by ArchDuque-Pancake** [comment]
I don't know what happened. Wifi got slow and now it sems I created something?

File: android/assets/jsons/translations/Spanish.properties [hunk 1]
```diff
@@ -5342,7 +5342,7 @@ Provides [stats] [cityFilter] per turn = proporciona [stats] [cityFilter] por tu
 Provides [amount] Happiness = proporciona [amount] Felicidad
 Provides military units every ≈[amount] turns = Proporciona unidades militares cada ≈[amount] turnos
 Provides a unique luxury = Proporciona un lujo único.
-Cannot build [baseUnitFilter] units = No se pueden construir unidades [baseUnitFilter]
+Cannot build [baseUnitFilter] units = No se pued construír unidades [baseUnitFilter]
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:5345]: ¿?

**Comment by SomeTroglodyte** [comment]
> it seems I created something

Called god mode. No, seriously: "This commit has no content." - wouldn't worry about it.

**Review by SomeTroglodyte — APPROVED** [review]

File: android/assets/jsons/translations/Spanish.properties (hunk 1 above)
Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties:5345]: oops

**Pull request merged**